ultraThreshold.py

```diff
@@ -137,6 +142,7 @@
         scanData = getUltraScanResults(ohboard, options.gtx, THRESH_MAX - THRESH_MIN + 1, options.debug)
         sys.stdout.flush()
         for i in range(0,24):
+            if (mask >> i) & 0x1: continue
```

Comment:
Similarly to above comment regarding ultraScurve.py, if line is commented out generates exception. If line is uncommented gives expected behavior.

Conclude line is needed.